### PR TITLE
DEV-14724 - Adds variable for setting ARN partition

### PR DESCRIPTION
**Pull request by souza-dan**
https://datatamr.atlassian.net/browse/DEV-14724

### CODE REVIEW

**Comment by tamr-teamcity** [comment]
This PR doesn't appear to be linked to a DevOps/SRE jira ticket

**Comment by tamr-teamcity** [comment]
This PR doesn't appear to be linked to a DevOps/SRE jira ticket

**Comment by tamr-teamcity** [comment]
This PR doesn't appear to be linked to a DevOps/SRE jira ticket

File: examples/static-hbase/README.md [hunk 1]
```diff
@@ -3,13 +3,16 @@ This example depends on an existing VPC and subnet with configurations that meet
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 ## Requirements
 
-No requirements.
+| Name | Version |
+|------|---------|
+| aws | 3.27.0 |
```
Comment (daniosim) [inline examples/static-hbase/README.md:8]: I'm curious as to what changed to make these requirements appear?

**Review by daniosim — APPROVED** [review]
Looks good 👍

File: examples/static-hbase/README.md (hunk 1 above)
Comment (souza-dan) [reply examples/static-hbase/README.md:8]: oh, maybe it's because I've run the examples before and I have a .terraform folder here.

**Pull request merged**